<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Agregar Receta</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:&quot;#222&quot;; color: &quot;#eee&quot;</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>611</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QPushButton" name="cancelar_receta_btn">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ccd1d1;
 color: #222;
 border-radius: 10px;
border: 2px solid #eee</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLineEdit" name="paciente_input">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:'#222'; 
color: '#eee'; 
padding: 4px;
border-radius: 10px;
border: 2px solid #eee</string>
      </property>
      <property name="placeholderText">
       <string>Paciente</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="diagnostico_input">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:'#222'; 
color: '#eee'; 
padding: 4px;
border-radius: 10px;
border: 2px solid #eee</string>
      </property>
      <property name="placeholderText">
       <string>Diagnóstico</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" alignment="Qt::AlignRight">
     <widget class="QPushButton" name="agg_receta_btn">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ccd1d1;
 color: #222;
 border-radius: 10px;
border: 2px solid #eee</string>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="medicamento_input">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:'#222'; 
color: '#eee'; 
padding: 4px;
border-radius: 10px;
border: 2px solid #eee</string>
      </property>
      <property name="placeholderText">
       <string>Medicamento</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="cantidad_input">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:'#222'; 
color: '#eee'; 
padding: 4px;
border-radius: 10px;
border: 2px solid #eee</string>
      </property>
      <property name="placeholderText">
       <string>Cantidad</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="doctor_input">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:'#222'; 
color: '#eee'; 
padding: 4px;
border-radius: 10px;
border: 2px solid #eee</string>
      </property>
      <property name="placeholderText">
       <string>Doctor</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="emision_input">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:'#222'; 
color: '#eee'; 
padding: 4px;
border-radius: 10px;
border: 2px solid #eee</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Emisión</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="error_lbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: 'red';</string>
   </property>
   <property name="text">
    <string>Complete todos los campos</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Agregar Receta:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
